static/app/views/performance/transactionSummary/transactionSpans/spanDetails/exclusiveTimeChart.tsx

```diff
@@ -69,6 +69,9 @@ export default function ExclusiveTimeChart(props: Props) {
     browserHistory.push(to);
   };
 
+  const getExclusiveTimeDisplayedValue = (value: string) =>
+    value.replace('exclusive', 'self');
```

Comment:
This can live as a helper function outside of the component

Reply:
done!